src/simplewallet/simplewallet.cpp

```diff
@@ -4527,6 +4529,39 @@ bool simple_wallet::get_tx_note(const std::vector<std::string> &args)
   return true;
 }
 //----------------------------------------------------------------------------------------------------
+bool simple_wallet::set_description(const std::vector<std::string> &args)
+{
+  // 0 arguments allowed, for setting the description to empty string
+
+  std::string description = "";
+  for (size_t n = 0; n < args.size(); ++n)
+  {
+    if (n > 0)
+      description += " ";
+    description += args[n];
+  }
+  m_wallet->set_description(description);
+
+  return true;
+}
+//----------------------------------------------------------------------------------------------------
+bool simple_wallet::get_description(const std::vector<std::string> &args)
+{
+  if (args.size() != 0)
+  {
+    fail_msg_writer() << tr("usage: get_description");
+    return true;
+  }
+
+  std::string description = m_wallet->get_description();
+  if (description.empty())
+    success_msg_writer() << "no description found";
+  else
+    success_msg_writer() << "description found: " << description;
```

Comment:
Here too (tbh, I'd just omit the "description found: ", which is wholly redundant, but meh).

Reply:
Ok, done. I share your feeling about the "description found: " prompt, which again came from the TX notes code, but I left it now. Maybe I will go through the CLI wallet as a whole later in a little "prompt improving project".